Add README for [example/Waspello] (fix #663) (#738)
wasp-lang/wasp: examples/waspello/README.md

Waspello 
=========

**Waspello** is a trello clone app made with Wasp.

This app is deployed at [https://waspello.netlify.app/](https://waspello.netlify.app/).

# Development

> ### Databse

Reviewer: ```suggestion
### Database
```

Waspello 
=========

**Waspello** is a trello clone app made with Wasp.

> This app is deploted at [https://waspello.netlify.app/](https://waspello.netlify.app/).

Reviewer 2: ```suggestion
This app is deployed at [https://waspello.netlify.app/](https://waspello.netlify.app/).
```

# Development

> ### Databse

Reviewer 2: If we are repeating this for every example app, do we already have a place in the docs where we could just link to?

Reviewer 2: This is the place in the docs explaining how to run your postgres db, so we can just link to it: https://wasp-lang.dev/docs/language/features#postgresql

Author: Yup that would be the best to do...

> Wasp needs postgre database running - provide it with database conncetion URL via env var `DATABASE_URL` - best to it via `.env` file.

Reviewer 2: ```suggestion
Wasp needs postgre database running - provide it with database connection URL via env var `DATABASE_URL` - best to do it via `.env` file.
```
